README.md

```diff
@@ -1,15 +1,65 @@
 # Getting Started with the .NET MAUI Maps Control
 
-A quick-start project that helps you to integrate a MAUI Maps control in a .NET MAUI application. This project contains code to load shapes from a remote file, bind a data source for a shape layer, load a map in a tile layer, and enable basic features like data labels, markers, legends, and tooltips.
- 
-Refer to the following documentation to learn about the .NET MAUI Maps Control: 
-https://help.syncfusion.com/maui/maps/getting-started
-
-Check out this online example of the .NET MAUI Maps Control:
-https://github.com/syncfusion/maui-demos/tree/master/MAUI/Maps
+The [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps?utm_source=vs_marketplace&utm_medium=listing&utm_campaign=maui-maps-github-samples) control is a powerful data visualization component that displays statistical information for a geographical area. A quick-start project that helps you to integrate a MAUI Maps control in a .NET MAUI application. This project contains code to load shapes from a remote file, bind a data source for a shape layer, load a map in a tile layer, and enable basic features like data labels, markers, legends, and tooltips.
```

Comment:
Use the below UTM Parameter, check in all places
(?utm_source=github&utm_medium=listing&utm_campaign=maui-maps-github-samples)

Reply:
updated